extensions/default/src/DicomWebDataSource/retrieveStudyMetadata.js

```diff
@@ -34,9 +35,12 @@ export function retrieveStudyMetadata(
     throw new Error(`${moduleName}: Required 'StudyInstanceUID' parameter not provided.`);
   }
 
-  // Already waiting on result? Return cached promise
-  if (StudyMetaDataPromises.has(StudyInstanceUID)) {
-    return StudyMetaDataPromises.get(StudyInstanceUID);
+  // Already waiting on result? Return cached promise. The StudyMetaDataPromises
+  // key is a combination of the client name and StudyInstanceUID, because OHIF
+  // can retrieve dicom series belonging to a study from different servers.
```

Comment:
I think this is misleading. Maybe we say something along 

`OHIF has the capability to retrieve series from the same study from different DICOM web servers.` ?